threedgrut_playground/utils/rng.py

```diff
@@ -26,78 +26,198 @@
 """
 
 UInt32Buffer = Union[torch.LongTensor, np.uint32]
-UINT32_MASK = 0xFFFFFFFF    # torch doesn't natively support uint32 so we force the overflow with a mask
+UINT32_MASK = 0xFFFFFFFF  # torch doesn't natively support uint32 so we force the overflow with a mask
 
 
 SOBOL_DIRECTIONS = [
```

Comment:
Not a big deal - but is there an annotation to ignore formatting here?

Reply:
Sounds good, I added `# fmt: off` and `#fmt: on` around the section you highlighted!